core-bundle/src/Security/Authentication/Token/TokenChecker.php

```diff
@@ -112,4 +138,30 @@ private function getToken(string $sessionKey): ?TokenInterface
 
         return $token;
     }
+
+    private function getTokenFromStorage(string $context): ?TokenInterface
+    {
+        $request = $this->requestStack->getMasterRequest();
+
+        if (!$this->firewallMap instanceof FirewallMap || null === $request) {
+            return null;
+        }
+
+        $config = $this->firewallMap->getFirewallConfig($request);
+
+        if (!$config instanceof FirewallConfig || $config->getContext() !== $context) {
+            return null;
+        }
+
+        return $this->tokenStorage->getToken();
+    }
+
+    private function getTokenFromSession(string $sessionKey)
```

Comment:
Return typehint or phpdoc missing

Reply:
Fixed in 4d0b52bef783333027ec22e292d23ce52dbf0d76